### PR TITLE
Change Ignored Prototypes remove upstream chem combinations

### PR DESCRIPTION
**Pull request by Aurallianz**
## About the PR
Removed upstream chemical combinations; this may cause issues; please report them.

## Why / Balance
Admin Issues

**Changelog**
:cl:
- remove: Removed upstream chemical combinations

### CODE REVIEW

**Comment by Rainbeon** [comment]
Mess Tech needs the drinks.yml reactions as many of them are shared between upstream and CM13

**Comment by Aurallianz** [comment]
This section should be removed. Most of our drinks are not upstream drinks. They are custom RMC drinks, so someone must add custom options for RMC if they need to be included, we should not be using the upstream ones.

**Comment by QuietlyWhisper** [comment]
Probably not a good idea unless someone wants to recreate the ENTIRE baystation chef content because I'm fairly certain they're just using their upstream's content for that

**Comment by QuietlyWhisper** [comment]
TLDR this will result in an extremely long delay for mess tech release if we don't make a compromise

**Comment by Aurallianz** [comment]
We could keep the drinks; the only issue is that many of those recipes don't work anyway, and if they did, we should probably copy them out and put them in a separate rmc-controlled drinks.yml file. I didn't want to go through and remove each recipe individually and then check to see if it worked, mostly because I do not like screwing around with upstream files.

**Comment by Aurallianz** [comment]
I was also working on mess-tech, but someone else took it over. I'm not sure who it was, but they should definitely be auditing this list then.
